mkdocs.yml

```diff
@@ -86,7 +86,8 @@ extra:
     - icon: fontawesome/brands/docker
       link: https://hub.docker.com/orgs/cmsml/repositories
     - icon: fontawesome/solid/comment-dots
-      link: https://hypernews.cern.ch/HyperNews/CMS/get/machine-learning.html
+      link: https://cms-talk.web.cern.ch/c/physics/ml/104
+#      link: https://hypernews.cern.ch/HyperNews/CMS/get/machine-learning.html
```

Comment:
Please remove this comment. It's no longer a valid link.

Reply:
Will do 👍